packages/rule-components/src/index.js

```diff
@@ -0,0 +1 @@
+console.log('rule-components');
```

Comment:
console.log

Reply:
That's actually needed, because we don't have the component ready, yet.